### ISSUE TITLE
Not clear enough update message

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaLang/juliaup#55]
In playing a more naive user, I did the following when the program first lanuched:
```julia
The latest version of Julia in the 1 channel is Julia 1.6.1. You currently have Julia 1.5.3 installed. Run:

  juliaup update

to install Julia 1.6.1 and update your current channel to that version.
┌ Warning: Expected non-existent C:\Users\densb\AppData\Local\Programs\Julia 1.5.3\share\julia to be your Julia directory.
│ Certain functionality will be disabled.
│ To fix this, try deleting Revise's cache files in ~/.julia/compiled/v1.5/Revise, then restart Julia and load Revise.
│ If this doesn't fix the problem, please report an issue at https://github.com/timholy/Revise.jl/issues.
└ @ Revise C:\Users\densb\.julia\packages\Revise\VxkZO\src\packagedef.jl:1311
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.5.3 (2020-11-09)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> juliaup update
ERROR: syntax: extra token "update" after end of expression
Stacktrace:
 [1] top-level scope at none:1
```

The message should be more clear on the fact that this command should be put into the run window, or directly into a command prompt. I imagine that the uses who install julia with the app store will be the least technically savvy, so I think this is needed.

**Comment by davidanthoff** [comment]
I think this can be closed as a duplicate of https://github.com/JuliaLang/juliaup/issues/43, right?

**Comment by KronosTheLate** [comment]
> I think this can be closed as a duplicate of #43, right?

The issue is the same, and the solution proposed is more elegant. Good close. Implementing the semicolon tip would be lovely
